### ISSUE TITLE
Gesture typing does not support contractions

### ISSUE THREAD

**Issue by 12people** [AnySoftKeyboard/AnySoftKeyboard#2163]
### Steps to reproduce
1. Try to type a contraction using gesture typing (= swipe). "It's" or "they're", for example.

### Expected behaviour
The contraction should be typed or, at the very least, among the suggested words.

### Actual behaviour
No contractions are typed nor among the suggested words.

**Android OS version:**
Oreo

**Device manufacturer and model:**
Fairphone 3

**List of installed add-ons (like languages, or themes):**
Czech for ASK

**Comment by decadent** [comment]
Contractions appear and are typed if you include the apostrophe in the sequence. It's not too intuitive, though, and other apps allow to omit the apostrophes.

**Comment by khimaros** [comment]
i am having a similar issue with `haven't`

**Comment by TheShanMan** [comment]
Huge limitation IMO. I just tried swiping through the apostrophe and it didn't seem to work for me. But apostrophes shouldn't be required for the contraction to be recognized or at least show up as a top suggestion.

**Comment by menny** [comment]
so, when swiping `don't`, you do not swipe through `'`?

**Comment by TheShanMan** [comment]
That's correct. That's how gboard swiping works. Perhaps swiping is linked to auto-correct because if you type "dont" it'll auto-correct to "don't". IMO it would feel weird to swipe through the apostrophe. I would _love_ swiping to work effectively. Moving from gboard to ASK was frustrating enough with regard to swiping, that I've since learned to make due with double thumb typing instead. It's slower, but less frustrating than ASK swiping. If swiping ever gets significantly improved, I'd love to give it another try! Thanks for considering this!

**Comment by zanovis** [comment]
This seems fixed now, no?
